include/boost/geometry/util/math.hpp

```diff
@@ -191,6 +192,66 @@ struct square_root<T, true>
 };
 
 
+
+template
+<
+    typename T,
+    bool IsFundemantal = boost::is_fundamental<T>::value /* false */
+>
+struct modulo
+{
+    typedef T return_type;
+
+    static inline T apply(T const& value1, T const& value2)
+    {
+        // for non-fundamental number types assume that sqrt is
```

Comment:
Thanks for catching this. Typo introduced by careless copy/paste....
Will fix it.